docs/src/explanations/theming/themes.md

```diff
@@ -130,6 +130,22 @@ ggplot_theme = Theme(
 with_theme(example_plot, ggplot_theme)
 ```
 
+LScene blocks can be themed through an Axis3D block theme. 
+
+Here is how to define new default axisnames:
+
+    Default_name_theme = Axis3D = (; 
+            names = 
+                (; axisnames = ("lon", "lat", "alt")
+            )
+        )
+    )
+    
+An exhaustive collection of the axis attribute can be found with :
+
+    LScene(Figure()).scene[OldAxis].attributes
```

Comment:
Ah yes my bad I tried ?Axis3D instead of ?axis3d. Also I thought it would be useful to put the OldAxis thing in the documentation but that's probably another issue/PR.